dubbo-admin-backend/pom.xml

```diff
@@ -110,6 +110,12 @@
             <artifactId>netty-all</artifactId>
         </dependency>
 
+        <dependency>
+            <groupId>org.apache.curator</groupId>
+            <artifactId>curator-test</artifactId>
+            <version>4.1.0</version>
```

Comment:
extract the version property

Reply:
I'll remove this dependency since it is include in the parent `pom.xml`

Reply:
I'll remove this dependency since it is include in the parent `pom.xml`